api/k8sconsts/odiglet.go

```diff
@@ -17,6 +17,12 @@ const (
 	OdigletOSSInstalledLabel        = "odigos.io/odiglet-oss-installed"
 	OdigletEnterpriseInstalledLabel = "odigos.io/odiglet-enterprise-installed"
 	OdigletInstalledLabelValue      = "true"
+
+	// ConfigMap used to store custom/updated Go instrumentation offsets
+	GoOffsetsPublicURL = "https://storage.googleapis.com/odigos-cloud/offset_results_min.json"
+	GoOffsetsConfigMap = "odigos-go-offsets"
+	GoOffsetsFileName  = "offset_results.json"
```

Comment:
should we have the word "go" in this file name to make it obvious that these offsets are only for golang? so in the future if we add additional offsets things are better structured.

Reply:
Sure, we should probably update the published file name too but we can do that